hydroflow_lang/src/graph/ops/source_stream.rs

```diff
@@ -36,6 +39,11 @@ pub const SOURCE_STREAM: OperatorConstraints = OperatorConstraints {
     is_external_input: true,
     ports_inn: None,
     ports_out: None,
+    properties: FlowProperties {
+        deterministic: FlowPropertyVal::No,
```

Comment:
Should probably be `CodeBlock` or `Preserve` in the sense that the properties of determinism and monotonicity come from the sources that this is streaming.